components/x-teaser/__tests__/__snapshots__/snapshots.test.js.snap

```diff
@@ -202,7 +202,7 @@ exports[`x-teaser / snapshots renders a Hero teaser with packageItem data 1`] =
         data-trackable="teaser-tag"
         href="#"
       >
-        Financial crisis: Are we safer now? 
+        Financial crisis: Are we safer now?
```

Comment:
these might have been removed by autoformatting or something? i think they're supposed to be there because that's the markup that's actually generated. tenner says the snapshot tests fail

Reply:
Yep I'm on this